tests/integration/test_emergency_contacts.py

```diff
@@ -47,16 +48,22 @@ def test_emergency_contacts_POST(client, auth_headers):
   }
 
   response = client.post(endpoint, json=newContact, headers=auth_headers["admin"])
-  assert is_valid(response, 401) # UNAUTHORIZED - An emergency contact with this name already exists
+  assert is_valid(response, 401) # UNAUTHORIZED - Emergency Contact With This Name Already Exists
+  assert response.json == \
+          {'message':
+           'An emergency contact with this name already exists'}
```

Comment:
This sounds like a validation issue rather than an authorization issue. Created issue #374 to address this